packages/lit-dev-content/samples/tutorials/working-with-lists/04/before/my-element.ts

```diff
@@ -0,0 +1,50 @@
+import {html, css, LitElement} from 'lit';
+import {customElement} from 'lit/decorators.js';
+// TODO: Import directives
+
+@customElement('my-element')
+class MyElement extends LitElement {
+  static styles = css`
+    /* playground-fold */
+    :host {
+      display: block;
+      width: 400px;
+      height: 400px;
+    }
+    #board {
+      display: grid;
+      grid-template-columns: repeat(8, 12.5%);
```

Comment:
Can we remove the hard-coded `8`'s here and elsewhere in this module? It should either be a `const size = 8` at the top of the module or settable in the render method and passed to css via a custom property. I also think this can just be `1fr` instead of `12.5%`.

Reply:
I agree that as a finished product, I'd like to not hardcode the 8. Making it settable and passing into css via custom property sounds great. I think that would be great as a playground example too.

But as a tutorial step, I feel that may be too much. I initially had them as variables but thought it added extra noise in the instruction having to say `rows` by `columns` size board, instead of a fixed 8 by 8 board, and especially for the paragraph under the code sample that mentions `range()` generating integers from 0 to 7. Elliott has already mentioned concerns about this section having a lot going on. What do you think?

Perhaps it can be mentioned as extra credit for this step to make it more versatile. @justinfagnani and @e111077 have been talking about a way to add optional extra steps to these tutorials. Or maybe linking out to a playground example for a solution to the extra credit?

Agreed on the `1fr` which would work even if the size was dynamic.